### PR TITLE
Docs: Separate sentences

### PR DESCRIPTION
**Pull request by eaba**

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
Thanks for the tip @sean-gilliam

**Review by Aaronontheweb — APPROVED** [review]
LGTM

File: docs/articles/clustering/distributed-publish-subscribe.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ You register actors to the local mediator with `DistributedPubSubMediator.Subscr
 
 You publish messages by sending `DistributedPubSubMediator.Publish` message to the local mediator.
 
-Topic actors are automatically removed from the registry when they are terminated; they are terminated if there are no subscribers for the duration configured by `akka.cluster.pub-sub.removed-time-to-live`, which defaults to 2 minutes. You can change the deadline by setting `removed-time-to-live` to a custom duration.
+Topic actors are automatically removed from the registry when they are terminated. Termination occurs when there are no subscribers for the duration configured by `akka.cluster.pub-sub.removed-time-to-live`, which defaults to 2 minutes. You can change the deadline by setting `removed-time-to-live` to a custom duration.
```
Comment (eaba) [inline docs/articles/clustering/distributed-publish-subscribe.md:33]: Any further improvement here? @sean-gilliam @Aaronontheweb

Comment (sean-gilliam) [reply docs/articles/clustering/distributed-publish-subscribe.md:33]: LGTM

**Pull request merged**